default.sh

```
@@ -308,15 +388,14 @@ onboard_mock_relying_party_with_mock_rp_oidc_client(){
 	--folder partner_request_mapping_to_policyname \
 	--folder approve-partner-mapping-to-policy \
 	--folder get-jwks \
-	--folder keycloak-authentication-for-mock-plugin \
 	--folder create-oidc-client \
 	--folder create-oidc-client-through-esignet \
 	--folder delete-user \
     $ADD_SSL_NEWMAN \
-    --export-environment ./config-secrets.json -d ./default-mock-rp-oidc-policy.json -r cli,htmlextra --reporter-htmlextra-export "$reports_dir/mock-rp-oidc.html" --reporter-htmlextra-showEnvironmentData
+    --export-environment ./config-secrets.json -d "$POLICY_DATA_FILE" -r cli,htmlextra --reporter-htmlextra-export "$reports_dir/mock-rp-oidc.html" --reporter-htmlextra-showEnvironmentData
 privateandpublickeypair=$(jq -r '.values[] | select(.key == "privateandpublickeypair") | .value' config-secrets.json)
 privateandpublickeypair=$(echo -n "$privateandpublickeypair" | base64)
```

Shellcheck (0.11.0)

[warning] In POSIX sh, echo flags are undefined.

(SC3037)